cupy/core/_routines_statistics.pyx

```diff
@@ -338,6 +348,50 @@ cdef _mean = create_reduction_func(
      'out0 = a / _type_reduce(_in_ind.size() / _out_ind.size())', None))
 
 
+_count_non_nan = create_reduction_func(
+    'cupy_count_non_nan',
+    ('e->e', 'f->f', 'd->d'),
```

Comment:
Why not return an integer?

Reply:
Previously, I was getting `dtype` mismatch. Now resolved.